src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -953,4 +953,7 @@
   <data name="PowerLauncher_EnablePluginToggle.AutomationProperties.Name" xml:space="preserve">
     <value>Enable plugin</value>
   </data>
+  <data name="PowerLauncher_NotAccessibleWarning.Text" xml:space="preserve">

```

Comment:
I know we use PowerLauncher on the other strings. But because we want to move away we shouldn't create new place with this wording.

@enricogior 
Is `PowerToysRun` the correct wording?

Reply:
@mykhailopylyp 
let's stop using `PowerLauncher` or `Launcher` for any new id/variable/etc.
Even if this introduces an inconsistency, we don't care since we will replace all `PowerLauncher` instances in the near future.

Reply:
done